test/spec/modeler/Linting.spec.js

```diff
@@ -175,13 +175,21 @@ describe('Linting', function() {
         container.innerHTML = '';
 
         reports.map((report) => {
-          const { id, message, category, rule } = report;
+          const { id, message, category, rule, documentation } = report;
 
           if (category === 'rule-error') {
             return domify(`<div class="errorItem">Rule error: Rule <${ escapeHTML(rule) }> errored with the following message: ${ escapeHTML(message) }</div>`);
           }
 
-          const element = domify(`<div class="errorItem">${ id }: ${escapeHTML(message) }</div>`);
+          const element = domify(`<div class="errorItem">${ id }: ${escapeHTML(message) } </div>`);
```

Comment:
A more secure way to handle this would be:

```suggestion
          const element = domify('<div class="errorItem" />');
          element.textContent = `${ id }: ${ message }`
```

We don't need to rely on `escapeHTML`'s correctness in such cases.

Reply:
Could be good, but would prevent us from styling (cf. [later changes](https://github.com/camunda/linting/pull/100/commits/8e79e03e35db14e26782fdf0779eaf2e76f0c1cb)).

Reply:
Oh ok, so we actually don't escape all of the HTML but rather allowlist tags.

Reply:
Yes.